src/dali/plugin/input/rest/binance_com.py

```diff
@@ -0,0 +1,1295 @@
+                    timestamp=self._rp2timestamp_from_ms_epoch(transaction[_TIMESTAMP]),
+                    asset=out_asset,
+                    exchange=self.__BINANCE_COM,
+                    holder=self.account_holder,
+                    transaction_type=Keyword.SELL.value,
+                    spot_price=Keyword.UNKNOWN.value,
+                    crypto_out_no_fee=str(crypto_out_no_fee),
+                    crypto_fee=str(crypto_fee),
+                    crypto_out_with_fee=str(crypto_out_with_fee),
+                    fiat_out_no_fee=None,
+                    fiat_fee=None,
+                    notes=(
+                        f"{notes + '; ' if notes else ''} Sell side of conversion from "
+                        f"{conversion_info}"
+                        f"({in_asset} in-transaction unique id: {transaction[_ID]}"
+                    ),
+                )
+            )
+
+    def _process_transfer(self, transaction: Any, intra_transaction_list: List[IntraTransaction]) -> None:
+
+        # This is a CCXT list must convert to string from float
+        amount: RP2Decimal = RP2Decimal(str(transaction[_AMOUNT]))
+
+        if transaction[_TYPE] == _DEPOSIT:
+            intra_transaction_list.append(
+                IntraTransaction(
+                    plugin=self.__BINANCE_COM,
+                    unique_id=transaction[_TXID],
+                    raw_data=json.dumps(transaction),
+                    timestamp=transaction[_DATETIME],
+                    asset=transaction[_CURRENCY],
+                    from_exchange=Keyword.UNKNOWN.value,
+                    from_holder=Keyword.UNKNOWN.value,
+                    to_exchange=self.__BINANCE_COM,
+                    to_holder=self.account_holder,
+                    spot_price=Keyword.UNKNOWN.value,
+                    crypto_sent=Keyword.UNKNOWN.value,
+                    crypto_received=str(amount),
+                )
+            )
+        elif transaction[_TYPE] == _WITHDRAWAL:
+            intra_transaction_list.append(
+                IntraTransaction(
+                    plugin=self.__BINANCE_COM,
+                    unique_id=transaction[_TXID],
+                    raw_data=json.dumps(transaction),
+                    timestamp=transaction[_DATETIME],
+                    asset=transaction[_CURRENCY],
+                    from_exchange=self.__BINANCE_COM,
+                    from_holder=self.account_holder,
+                    to_exchange=Keyword.UNKNOWN.value,
+                    to_holder=Keyword.UNKNOWN.value,
+                    spot_price=Keyword.UNKNOWN.value,
+                    crypto_sent=str(amount),
+                    crypto_received=Keyword.UNKNOWN.value,
+                )
+            )
+        else:
+            self.__logger.error("Unrecognized Crypto transfer: %s", json.dumps(transaction))
+
+    def _process_withdrawal(self, transaction: Any, out_transaction_list: List[OutTransaction], notes: Optional[str] = None) -> None:

```

Comment:
This seems to refer to fiat withdrawals only: should we rename this _process_fiat_withdrawal?